packages/analytics/src/integrations/Omnitracking/__tests__/Omnitracking.test.ts

```diff
@@ -527,6 +527,44 @@ describe('Omnitracking', () => {
       );
     });
 
+    describe('Site Performance', () => {
+      it('should not track an Site Performance event if the required parameters are missing', async () => {
+        const data = generateTrackMockData({
+          event: EventType.SitePerformance,
+          properties: {
+            performanceStats: undefined,
+          },
+        });
+
+        await omnitracking.track(data);
+
+        expect(mockLoggerError).toHaveBeenCalledWith(
+          expect.stringContaining(
+            'to track this event, should be added a valid "performanceStats" property.',
+          ),
+        );
+        expect(postTrackingSpy).not.toHaveBeenCalled();
+      });
+
+      it('should not track an Site Performance event if the required parameters has wrong type', async () => {
```

Comment:
Same as above. Also, can we have a test with the happy path, where the correct value is passed?

Reply:
Done